### PR TITLE
samson in readme

### PR DESCRIPTION
**Pull request by sandlerr**
:koala: 

This is going to get copied and pasted for a bunch of apps.

/cc @zendesk/quokka

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,3 +16,8 @@ If the optional setting "partyPooper" is checked that will tone down the vulgari
 ##### Look for this in the Zendesk Apps Marketplace
 
 ![](http://i.imgur.com/bixQCt5.png)
+
+##Deployment
```
Comment (adammw) [inline README.md]: Missing space?

**Comment by adammw** [comment]
:+1:

File: README.md (hunk 1 above)
Comment (sandlerr) [inline README.md]: markdown doesn't render that whitespace, do you want it for the sake of md code style?

Comment (adammw) [inline README.md]: yep

**Comment by itirkarp** [comment]
:+1:

**Comment by adammw** [comment]
:+1:

**Comment by atroche** [comment]
:+1:

**Pull request merged**